csrc/runtime/executor.cpp

```diff
@@ -1266,6 +1266,7 @@ KernelArgumentHolder KernelExecutor::run(
           stream,
           executor_entry->arg_ptrs.data()));
     }
+    NVFUSER_CUDA_SAFE_CALL(cuCtxSynchronize());
```

Comment:
Why is this necessary?

Reply:
I am trying to locate the actual location of the first error. Not sure why I can't reproduce the err locally even with compute-sanitizer. CI reported error `INTERNAL ASSERT FAILED at "/opt/pytorch/nvfuser/csrc/runtime/executor_utils.cpp":544, please report a bug with repro script to NVFuser at https://github.com/NVIDIA/Fuser/issues. CUDA error: CUDA_ERROR_ILLEGAL_ADDRESS failed with error an illegal memory access was encountered` at:
```
CudaExecutable::~CudaExecutable() {
  if (module != nullptr) {
    NVFUSER_CUDA_SAFE_CALL(cuModuleUnload(module));
    module = (CUmodule)0x2a2a2a2a2a2a2a2a;
  }
}
```